phpseclib/Crypt/Base.php

```diff
@@ -2001,11 +2057,48 @@ function _createInlineCryptFunction($cipher_code)
      * for which $mode the lambda function was created.
      *
      * @access private
-     * @return &Array
+     * @return Array &$functions
      */
     function &_getLambdaFunctions()
     {
         static $functions = array();
         return $functions;
     }
+
+    /**
+     * Generates a digest from $bytes
+     *
+     * @see _setupInlineCrypt()
+     * @access private
+     * @param $bytes
+     * @return String
+     */
+    function _trapdoor($bytes)
+    {
+        if (!defined('CRYPT_BASE_WHIRLPOOL_AVAILABLE')) {
```

Comment:
phpseclib / php4 / 1.0 already uses non-constant variables. eg. MATH_BIGINTEGER_OPENSSL_ENABLED, MATH_BIGINTEGER_MODE, CRYPT_RSA_MODE, etc.

For php5 / 2.0, a static class variable very definitely ought to be used but static variables aren't available in php4 / 1.0 and I don't think a property is a good way to go on this. I mean, if we go with a property on this for php4 / 1.0 we should probably go with properties for all the above instances where constants are already being used in this manner and even if we omitted `MATH_BIGINTEGER_*` it's still a check that'd now have to be done over and over and over again when instead it could have been done once and just once and that's it.